mlir/lib/IR/AffineExpr.cpp

```diff
@@ -237,6 +239,144 @@ unsigned AffineDimExpr::getPosition() const {
   return static_cast<ImplType *>(expr)->position;
 }
 
+/// Returns the divide type which are as follow floorrdiv = 1, ceildiv = 2,
+/// and otherwise 0.
+static int getDivideType(AffineExprKind exprKind) {
+  if (exprKind == AffineExprKind::FloorDiv)
+    return 1;
+  if (exprKind == AffineExprKind::CeilDiv)
+    return 2;
+  return 0;
```

Comment:
Mapping this makes things unreadable at the use sites. Use `enum` or use `AffineExprKind` itself.